docs/basics/101-180-FAQ.rst

```diff
@@ -411,6 +411,21 @@ dataset.
 When using the terminal command ``cp`` [#f1]_, it is sufficient to use the
 ``-L``/``--dereference`` option. This will follow symbolic links, and make
 sure that content gets moved instead of symlinks.
+Remember that if you are copying some annexed content out of a dataset without
+unlocking it first, you will only have "read" access on the files you have just 
+copied. Therefore you can :
+- either unlock the files before copying them out, 
+- or copy them and then use the command ``chmod`` to be able to edit the file.
+
+```
+# this will give you 'write' permission on the file
+chmod +w filename
+```
+.. todo::
+
+   add a link to very simple tuto on chmod
```

Comment:
Maybe: https://bids.github.io/2015-06-04-berkeley/shell/07-perm.html?

Reply:
Sorry for leaving this one so long on the back burner: I seemed to have missed those ntofications.

Will add the link now.